### PR TITLE
Add missed Azure region values

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**

### CODE REVIEW

**Comment by yaohaizh** [comment]
@weidongxu-microsoft please associate with the issue if possible when create PR for better tracking and provide meaningful PR title. Thus, we can get track the history and commit message. Thanks.

**Comment by qub1n** [comment]
This issue 
https://github.com/Azure/azure-libraries-for-net/issues/796

**Comment by weidongxu-microsoft** [comment]
image

eh.. so I am the only one who can see this?

File: Tests/Fluent.Tests/ResourceManager/SubscriptionsTests.cs [hunk 1]
```diff
@@ -65,5 +68,31 @@ public async Task CanGetByIdAsync()
                 Assert.Equal(subscription.SubscriptionId.ToLowerInvariant(), azure.SubscriptionId.ToLowerInvariant());
             }
         }
+
+        [Fact(Skip = "Util to generate missing regions")]
+        public void GenerateMissingRegion()
```
Comment (yaohaizh) [inline Tests/Fluent.Tests/ResourceManager/SubscriptionsTests.cs]: Should we skip this test case?

Comment (weidongxu-microsoft) [reply Tests/Fluent.Tests/ResourceManager/SubscriptionsTests.cs]: It is actually util code to faciliate generating code when azure should add new region in the future.

It is no use in replay mode, it had to run in live. Hence the SKIP.

**Pull request merged**